app/models/smart_proxy.rb

```diff
@@ -88,22 +86,25 @@ def has_feature?(feature)
     self.features.any? { |proxy_feature| proxy_feature.name == feature }
   end
 
+  def features_to_lowercase
+    features.pluck("LOWER(REPLACE(name, ' ', '_'))").uniq
```

Comment:
It is needed to load the erb files.
The feature is `DHCP`, the erb file is `_dhcp.html.erb`.
It comes more handy in places like `Puppet CA`, where the erb file should be `_puppet_ca.html.erb`